src/Okta.Sdk/Generated/Error.Generated.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="Error.Generated.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+// This file was automatically generated. Don't modify it directly.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Okta.Sdk.Internal;
+
+namespace Okta.Sdk
+{
+    /// <inheritdoc/>
+    public sealed partial class Error : Resource, IError
```

Comment:
Why do we have an error model in the spec? Is this expected @royalchan-okta @gormanho-okta?